README.md

```diff
@@ -39,21 +39,6 @@ Leveraging PostgreSQL for the database, Supabase streamlines backend development
 - **Authentication:** Provides secure user authentication with seamless integration for frontend frameworks like React.
 - **Serverless Functions:** Enables scalable and cost-effective backend operations without managing servers.
 
-### Testing
-**Set Up**
-Before running tests you will need to set up your authentication. An `example.auth.setup.ts` file is provided in the repo.
-
-Open this file and follow the indications in comments. You must have signed up to Kindly with a test account before this, as you will need to fill in your credentials.
-
-Add your test user details to your `.env` variables. These are described in the `.env.example`.
-<hr>
-
-Tests for this codebase use Playwright. Once fully set up, you can run all tests using the command `npm run ci`
-
-This command has been set up with `start-server-and-test` npm package to spin up the local server ahead of running end-to-end tests. Unit tets for components are also run in the same script by using the experimental playwright-ct package.
-
-Individual tests can be run using `npx playwright test -c playwright-ct.config.ts /pathToTestFile`
-
 ### Other: Nodemailer
```

Comment:
So that section is actually talking about the dependency that handles automatic email notifications to the users, which Shaughn has re-instated recently.

But, I have made an aesthetic change that I think will make the section a bit clearer and maintainable as we add other packages:

> ### Other
> 
> **Nodemailer:**
> A Node.js library for sending emails programmatically, Nodemailer enhances communication within the application, facilitating notifications and updates.